docs/deployment/pull-backup.rst

```diff
@@ -0,0 +1,137 @@
+.. include:: ../global.rst.inc
+.. highlight:: none
+
+Backing up in pull mode
+=======================
+
+Assuming you have a pull backup system set up with borg, where a backup server
+pulls the data from the target via sshfs. In this mode, the backup client's file
+system is mounted remotely on the backup server. Pull mode is even possible if
+the SSH connection must be established by the client via a remote tunnel. Other
+network file systems like NFS or SMB could be used as well, but sshfs is very
+simple to set up and probably the most secure one.
+
+There are some restrictions caused by sshfs. For example, unless you define UID
+and GID mappings when mounting via sshfs, owners and groups of the mounted file
+system will probably change, and you may not have access to those files if
+BorgBackup is not run with root privileges.
+
+The sshfs is a FUSE file system and uses the SFTP protocol, so there may be also
+other unsupported features that the actual implementations of sshfs, libfuse and
+sftp on the backup server do not support, like file name encodings, ACLs, xattrs
+or bsdflags. So there is no guarantee that you are able to restore a system
+completely in every aspect from such a backup.
+
+.. warning::
+
+    To mount the client's root file system you will need root access to the client.
+    This contradicts to the usual threat model of BorgBackup, where clients don't
+    need to trust the backup server (data is encrypted). In pull mode the server
+    (when logged in as root) could cause unlimited damage to the client. Therefore,
+    pull mode should be used only from servers you do fully trust!
+
+Creating a backup
+-----------------
+
+In this approach the client file system is simply mounted and then backed up.
+Note that the backup is created from within the mount point so that all files
+in the archive have their original paths (otherwise they would be backed up with
+the mount point prefix, e.g. /mnt/sshfs/bin/bash instead of /bin/bash).
+
+::
+
+    sshfs root@host:/ /mnt/sshfs
+    cd /mnt/sshfs
+    borg create /path/to/repo::archive . # note the dot!
```

Comment:
this is still storing wrong user/group names into the archive.

either: do not store names (just numeric ids)

or: work in a way so the id-to-name mapping of the backupped system is active.

Reply:
You're right, that's a point I didn't consider yet. So again a chroot might be the best solution here? Only storing numeric IDs would make the partial restore with mapping impossible.

Reply:
yeah, guess it is chroot time again.